### ISSUE TITLE
EVERYONE HELP ME PLZ

### ISSUE THREAD

**Issue by Taki21** [SWNetworkDevelopers/PHPStudio#1]
@MCUnderground 
@PocketKiller 
@Asif6474 
@remotevase 
@applqpak 
@Taki21 (LOL Dats ME)
@AmiroEmad 
@GrimDevs 
If you could, this is a big project so I would like some help with it!
Ok you all have read the ReadMe.MD Features I hope...
so...Here

@MCUnderground and @Taki21 and Whoever else
help with The Java and Folder Structure and XML thingy

We all help with little things **IF WE CAN!**

Thanks and hope you can help!

**Comment by remotevase** [comment]
As you saw me say on forums, I suck at java, but I might be able to help with the xml

**Comment by remotevase** [comment]
Also, I have no previous knowledge of application creating. I can only make websites, and ImagicalMine Plugins

**Comment by Taki21** [comment]
Lol xml go ahead xD

**Comment by Asif6474** [comment]
#I duck at java too

**Comment by Asif6474** [comment]
*suck

**Comment by Asif6474** [comment]
My dad tried teaching me but it failed badly ('-')

**Comment by Taki21** [comment]
XD @Asif6474

**Comment by Taki21** [comment]
AT least help on something asif xS

**Comment by Asif6474** [comment]
Which read.me

**Comment by Asif6474** [comment]
Ooh that one

**Comment by Asif6474** [comment]
Idk what xml is

**Comment by Asif6474** [comment]
WHAT R WE EVENTS DOIN? COMING WHAT ?????

**Comment by Asif6474** [comment]
*even

**Comment by MCUnderground** [comment]
I nub :(

**Comment by MCUnderground** [comment]
You need to.generate project using Eclipse or AndroidStudio and upload them

**Comment by PocketKiller** [comment]
fixed syntax error.

**Comment by AmiroEmad** [comment]
@taki21 sorry taki i didnt finish my java course yet i will try to finish it fast and start helping

**Comment by Taki21** [comment]
@AmiroEmad  Lol Im learning to in my handy dandy Java Book xD

**Comment by Taki21** [comment]
Lol pressing all deez random buttons...

**Comment by GrimDevs** [comment]
Sure I have a go but me pc dot. Like php7 it sucks can you help me make it back to php 5 like the old one plz I help o try Java something new I could try what possible could go wrong like If We put a Nucler Power Station And a Spider Breeding plz It is No Goud go possible go wrong can it Lol I give it try and I might have to wait a few weeks because ma house is not good and me floor is dropped so I have me house done but I get back to it ok taki

**Comment by GrimDevs** [comment]
What is the best Java course or should I do the same ways as with php just a try and error if code

**Comment by Taki21** [comment]
@GrimDevs CodeAcademy.org

**Comment by Taki21** [comment]
php5 still is COMPATIBLE with php7

**Comment by Taki21** [comment]
i personally think php7 is great

**Comment by GrimDevs** [comment]
thxs gtg at school

**Comment by Taki21** [comment]
@SWNetworkDevelopers/devs  Planning To Work On dis project this weekend!

**Comment by Taki21** [comment]
@MCUnderground 
@PocketKiller 
@Asif6474 
@remotevase 
@applqpak 
@Taki21 (LOL Dats ME)
@AmiroEmad 
@GrimDevs
@TheDragonRing 
Ok So I Uploaded Java Things! Lets get to work! You can contribute anytime you want btw and i dont mean 3 years later xD

**Comment by TheDragonRing** [comment]
I'll try but I know absolutely 0 percent Java though xD

**Comment by TheDragonRing** [comment]
I'll try out the www.codecademy.com out and see if I can learn anything from there

**Comment by Taki21** [comment]
lol xD I think we all need to learn (Almost all of us xD)

**Comment by TheDragonRing** [comment]
Yeah xD

**Comment by Taki21** [comment]
Yes! Learned Most Java Thingys xD

**Comment by Taki21** [comment]
OK!
Heres Da Plan:

@Taki21
@PocketKiller 
@MCUnderground 
@AmiroEmad 
@ Who ever knows java
- You guys and me will do the java and app functions

@remotevase 
@TheDragonRing 
@Taki21
@ Somebody Else
We work on XML stuff!

**Comment by MCUnderground** [comment]
If you wannq make app such like this,you need years of java knowledge... Plus that isnt android app folderstyle.. I dont know how to make suchthinglike this yet... I might skip it or help you on things i know

**Comment by MCUnderground** [comment]
I don't why you need xml coders? It is basic language... Which is markup kanguage.. Same as html(aka word languages)

**Comment by AmiroEmad** [comment]
wait who is going to pay for hosting the app on goolge play

for me i will host the app site

**Comment by TheDragonRing** [comment]
I have no idea.....

**Comment by AmiroEmad** [comment]
and how will we compile it to apk after coding so we can test

my last question are you sure that we can do it

**Comment by MCUnderground** [comment]
I dont think we can... And normally @Taki21 should edit normal project on his pc from there he will compile it.

**Comment by Taki21** [comment]
Im gonna host it

**Comment by Taki21** [comment]
Plus it depends how fast we learn and we all learn at different paces
and I have a very advanced book on advanced app making which provides some code
that can be useful! BTW Im lazy to do XML stuff

**Comment by Taki21** [comment]
@MCUnderground This is exact structure from AndroidStudio....

**Comment by Taki21** [comment]
Ok playsw.tk is working now

and

Do i need to upload all da Gradle Stuff?

**Comment by Taki21** [comment]
Im gonna delete this repo and upload when almost finished ...ok?
